src/extension.ts

```diff
@@ -37,7 +37,7 @@ function setupLSP() {
         // Options to control the language client
         const clientOptions: LanguageClientOptions = {
             // Register the server for racket documents
-            documentSelector: [{ scheme: "file", language: "racket" }, { scheme: "file", language: "fracas" }],
+            documentSelector: [{ scheme: "file", language: "racket" }],
```

Comment:
The racket language server seems to make `F12` slow, and frequently causes the extension to crash. I've disabled it for fracas files, but I will investigate how to restore document formatting support.

Reply:
👍 😢

Reply:
Maybe that's why there were two different sets of green triangle buttons too? Sorry about enabling this - I had thought we'd be able to get document formatting relatively cheaply. There's still a workaround of keeping Magic Racket installed, switching the file from "Fracas" to "Racket", formatting, then switching it back. That was what I was doing before the fracas plugin was working for `.frc` file formatting.

Reply:
The double green arrows were caused because I switched from being an "undefined" publisher to an official marketplace 
"Wonderstorm" publisher. Internally this seems to have generated a new unique ID for the plugin, which allowed it to be installed twice.